src/main/java/org/sonarlint/intellij/actions/IssuesViewTabOpener.java

```diff
@@ -96,4 +97,14 @@ public void showIssueDescription(LiveIssue liveIssue) {
   public void showIssueLocations(LiveIssue liveIssue) {
     showIssue(liveIssue, SonarLintIssuesPanel::selectLocationsTab);
   }
+
+  public void show(LocalHotspot localHotspot) {
+    openTab(SonarLintToolWindowFactory.TAB_HOTSPOTS);
+    ContentManager contentManager = getToolWindow().getContentManager();
+    Content content = contentManager.findContent(SonarLintToolWindowFactory.TAB_HOTSPOTS);
+    if(content.getComponent() instanceof SonarLintHotspotsPanel) {
```

Comment:
You have a sharp vision ! :)
It can't be something else indeed, I got inspired by similar code in the class. I will remove the check and directly cast the result.